libsolidity/inlineasm/AsmScope.h

```diff
@@ -85,6 +85,7 @@ struct Scope
 		Function(std::vector<JuliaType> const& _arguments, std::vector<JuliaType> const& _returns): arguments(_arguments), returns(_returns) {}
 		std::vector<JuliaType> arguments;
 		std::vector<JuliaType> returns;
+		boost::optional<LabelID> id;
```

Comment:
I agree. We already have exactly the same thing for labels, this is why I added it for now. The next refactoring should remove this.